pr_testing/test_multiple_prs.sh

```diff
@@ -786,6 +787,27 @@ if ! $CMSDIST_ONLY ; then # If a CMSSW specific PR was specified #
     exit 0
   fi
 
+  if [[ $PRODUCTION_RELEASE == "true" && ${PULL_REQUEST} == *"/cmssw#"* ]]; then
```

Comment:
Do we need quote in `*"/cmssw#"*`  ?

Reply:
It works with and without quotes.

Reply:
`*"/cmssw#"*` looks bit odd unless there is any recommendation to use quotes I would suggest to remove these